### PR TITLE
7031 BUG: Print Paper Service screen not displaying PDF preview

### PR DESCRIPTION
**Pull request by mrinsin**
image

- Fixed bug with no pdf url for paper service parties when uploading a document that is pending on creation
- Fixed bug with the way `isPendingOnCreation` was determining the value of pending
- Updated tests expect statement to reflect bug fix

### CODE REVIEW

File: web-client/integration-tests/docketRecordJourney.test.js [hunk 1]
```diff
@@ -1,8 +1,12 @@
-import { applicationContextForClient as applicationContext } from '../../shared/src/business/test/createTestApplicationContext';
+import {
+  applicationContextForClient as applicationContext,
+  applicationContextForClient,
+} from '../../shared/src/business/test/createTestApplicationContext';
```
Comment (rachaelparris) [inline web-client/integration-tests/docketRecordJourney.test.js]: Is there a reason to import this in two ways?

File: web-client/src/presenter/sequences/fileDocketEntrySequence.js [hunk 2]
```diff
@@ -61,16 +60,10 @@ const afterEntrySaved = [
...
   {
-    no: [
-      getIsPendingItemAction,
-      {
-        no: [completeDocketEntryQCAction],
-        yes: [],
-      },
-      caseDetailOrPrintPaperService,
-    ],
-    yes: [caseDetailOrPrintPaperService],
```
Comment (rachaelparris) [inline web-client/src/presenter/sequences/fileDocketEntrySequence.js:72]: It looks like this code was added to fix https://github.com/flexion/ef-cms/issues/6772 - can you verify that that is still functioning correctly with your fix?

File: web-client/integration-tests/docketRecordJourney.test.js (hunk 1 above)
Comment (rachelschneiderman) [reply web-client/integration-tests/docketRecordJourney.test.js]: Nope, thanks!

File: web-client/src/presenter/sequences/fileDocketEntrySequence.js (hunk 2 above)
Comment (mrinsin) [reply web-client/src/presenter/sequences/fileDocketEntrySequence.js:72]: Yes, this exact change was what caused the bug that we are fixing in this PR! A docket entry that has pending true (aka add to pending report is true) should still be QCd. We mobbed with Kristen on this, and also looked through  #6772  with her.

File: web-client/integration-tests/docketRecordJourney.test.js (hunk 1 above)
Comment (mrinsin) [reply web-client/integration-tests/docketRecordJourney.test.js]: Nope! Thanks for pointing it out!

**Review by rachaelparris — APPROVED** [review]

**Pull request merged**